splipy/BSplineBasis.py

```diff
@@ -304,6 +319,8 @@ def raise_order(self, amount):
             raise TypeError('amount needs to be a non-negative integer')
         if amount < 0:
             raise ValueError('amount needs to be a non-negative integer')
+        if amount == 0:
+            return self.clone()
```

Comment:
If you're wondering about this, it's because I wasn't able to `raise_order(0)` on a periodic basis, which I found it easier (for now) to single out the trivial case rather than fix.